syntax = "proto3";
message Msg901
{
    float msg_id = 1;
    int32 machine_id = 2;
    string msg = 3;
}